website/content/docs/agent/caching/persistent-caches/index.mdx

```diff
@@ -1,9 +1,71 @@
 ---
 layout: docs
-page_title: Vault Agent Persistent Cache Types
-description: Persistent Cache Types for Vault Agent Caching
+page_title: Vault Agent Persistent Caching
+description: Vault Agent Caching
 ---
 
-# Vault Agent Persistent Cache Types
+# Vault Agent Persistent Caching
+
+Vault Agent can restore tokens and leases from a persistent cache file created
+by a previous Vault Agent process. The persistent cache is a BoltDB file that
+includes tuples encrypted by a generated encryption key. The encrypted tuples
+include the Vault token used to retrieve secrets, leases for tokens/secrets, and
+secret values.
+
+-> **Note:** Vault Agent Caching persistent cache will only restore _leased_
```

Comment:
For the note, perhaps just call out the feature as Vault Agent Persistent Caching or Persistent Caching.

Reply:
Addressed in [eb64fb9](https://github.com/hashicorp/vault/pull/15195/commits/eb64fb945c55953276443c88fcc87f1a662f00a9)